src/propsheet/globals.h

```diff
@@ -35,6 +35,7 @@ extern BOOL gbEnumerateFaces;
 extern LONG gcxScreen;
 extern LONG gcyScreen;
 extern BOOL g_fForceV2;
+extern BOOL g_fLaunchedAsV2;

```

Comment:
You don't need this. The v2 console will declare itself as such in the structure passed in when the Property Sheet is created so you can key off of it.

Check `gpStateInfo->fIsV2Console` and see this for a sample where something is already keying to only show when launched from the V2 console: https://github.com/microsoft/terminal/blob/bac69f7cabec6068868034345e2aea22a9533696/src/propsheet/console.cpp#L579-L582

Reply:
How does this work for explorer-only invocation?

Reply:
I just attached to explorer and checked. `fIsV2Console` is false when we're in the standalone propsheet.

Reply:
My argument is then basically that either the old test was wrong or the new test is superfluous. 

I follow what you're saying and it's probably bad that explorer launch is detected as v1. So maybe change fIsV2Console to be the registry state, stop passing it from conhost v2, and key everything off of that one?